features/features.go

```diff
@@ -529,4 +529,12 @@ var (
 				productScope(ocpSpecific).
 				enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 				mustRegister()
+
+    FeatureGateIngressControllerLBSubnetsAWS = newFeatureGate("IngressControllerLBSubnetsAWS").
+                        reportProblemsToJiraComponent("Routing").
+                        contactPerson("gspence").
```

Comment:
nit: For network edge features, the contact person is @Miciah , but I don't know if there are rules about it having to be the team lead or not.